openshift-examples/keycloak-https-mutual-tls.json

```diff
@@ -184,7 +191,8 @@
                         "containers": [
                             {
                                 "name": "${APPLICATION_NAME}",
-                                "image": "jboss/keycloak-openshift",
+                                "image": "slaskawi/keycloak-jgroups-discovery",
```

Comment:
Do we need to change the image here?

Reply:
No! definitely not.... that one slipped in by an accident... fixing.